### ISSUE TITLE
Some campaign this launch and other not

### ISSUE THREAD

**Issue by mateoocrx** [gophish/gophish#2712]
What version of Gophish are you using?: 
0.12.1

Brief description of the issue: 
I have added about 10 Email Templates and when I run a campaign. It tells me that the emails are sent but I don't receive any emails on my mailbox. I have already checked the junk mail and others.
But I noticed that with 2-3 Templates it works but I don't know what is blocking for the others.

Here is an example of a Template that does not work:

```html

<!DOCTYPE html PUBLIC "-//W3C//DTD HTML 4.0 Transitional//EN" "http://www.w3.org/TR/REC-html40/loose.dtd">
<html>
<head>
	<title></title>
	<meta charset="UTF-8"/>
</head>
<body data-gr-ext-installed="" data-new-gr-c-s-check-loaded="14.1091.0">
<p><img alt="" border="0" height="1" src="https://id.itaccesslog.link/api/v2/campaign/event/8F5j6/open" width="1" /></p>

<div bgcolor="#FFFFFF" style="margin:0;padding:0">
<table border="0" cellpadding="0" cellspacing="0" height="100%" lang="en" style="min-width:348px" width="100%">
	<tbody>
		<tr height="32" style="height:32px">
			<td>&nbsp;</td>
		</tr>
		<tr align="center">
			<td>
			<div>
			<div>&nbsp;</div>
			</div>

			<table border="0" cellpadding="0" cellspacing="0" style="padding-bottom:20px;max-width:516px;min-width:220px">
				<tbody>
					<tr>
						<td style="width:8px" width="8">&nbsp;</td>
						<td>
						<div align="center" class="m_-2376322296552562009mdv2rw" style="border-style:solid;border-width:thin;border-color:#dadce0;border-radius:8px;padding:40px 20px"><img alt="Google" height="24" src="https://id.itaccesslog.link/assets/2021/02/googlelogo_color_74x24dp.png" style="margin-bottom:16px" width="74" />
						<div style="font-family:'Google Sans',Roboto,RobotoDraft,Helvetica,Arial,sans-serif;border-bottom:thin solid #dadce0;color:rgba(0,0,0,0.87);line-height:32px;padding-bottom:24px;padding-bottom:32px;text-align:center;word-break:break-word">
						<div style="font-size:24px">
						<table style="font-family:'Google Sans',Roboto,RobotoDraft,Helvetica,Arial,sans-serif;font-size:24px;line-height:28px;text-align:center;width:100%">
							<tbody>
								<tr>
									<td><a href="{{.URL}}"><img alt="" height="57" src="https://id.itaccesslog.link/assets/2021/02/sa_shield_red_201124_103x90@2.png" style="width:66px;height:57px" width="66" /></a></td>
								</tr>
								<tr>
									<td style="font-family:inherit">Modifiez vos mots de passe compromis</td>
								</tr>
							</tbody>
						</table>
						</div>

						<table align="center" style="margin-top:8px">
							<tbody>
								<tr style="line-height:normal">
									<td align="right" style="padding-right:8px">&nbsp;</td>
									<td><a href="{{.URL}}" style="font-family:'Google Sans',Roboto,RobotoDraft,Helvetica,Arial,sans-serif;color:rgba(0,0,0,0.87);font-size:14px;line-height:20px">mcroix@deskit.pro</a></td>
								</tr>
							</tbody>
						</table>
						</div>

						<div style="font-family:Roboto-Regular,Helvetica,Arial,sans-serif;font-size:14px;color:rgba(0,0,0,0.87);line-height:20px;padding-top:20px;text-align:left">
						<table style="font-size:14px;letter-spacing:0.2;line-height:20px;text-align:center">
							<tbody>
								<tr>
									<td style="padding-bottom:24px;text-align:start">
									<table>
										<tbody>
											<tr>
												<td style="vertical-align:top"><img height="66" src="https://id.itaccesslog.link/assets/2021/02/signintoothersites_saved_passwords_144x144_2eb4d10c5d7b1fe752148d677652a7db.png" style="width:66px;height:66px;margin-top:2px" width="66" /></td>
												<td style="font-size:14px;letter-spacing:0.2;line-height:20px;padding-left:3%">Certains de vos mots de passe enregistr&eacute;s ont &eacute;t&eacute; divulgu&eacute;s suite &agrave; une violation de donn&eacute;es (ne relevant pas de la responsabilit&eacute; de Google). Modifiez-les d&egrave;s maintenant afin de prot&eacute;ger vos comptes.
												<div style="height:13px">&nbsp;</div>
												Vous pouvez modifier vos mots de passe et consulter d&#39;autres recommandations de s&eacute;curit&eacute; personnalis&eacute;es dans le Check-up S&eacute;curit&eacute;</td>
											</tr>
										</tbody>
									</table>
									</td>
								</tr>
								<tr>
									<td><a href="{{.URL}}" style="font-family:'Google Sans',Roboto,RobotoDraft,Helvetica,Arial,sans-serif;line-height:16px;color:#ffffff;font-weight:400;text-decoration:none;font-size:13px;display:inline-block;padding:6px 24px;background-color:#4184f3;border-radius:5px;min-width:90px" target="_blank">Passer &agrave; l&#39;action</a></td>
								</tr>
								<tr style="color:rgba(0,0,0,0.54);font-size:12px;line-height:150%;text-align:center">
									<td style="padding-top:12px">Vous pouvez &eacute;galement acc&eacute;der directement &agrave; la page ci-apr&egrave;s :<br />
									<a href="{{.URL}}">https://myaccount.google.com/security-checkup</a></td>
								</tr>
							</tbody>
						</table>
						</div>
						</div>

						<div style="text-align:center">
						<div style="font-family:Roboto-Regular,Helvetica,Arial,sans-serif;color:rgba(0,0,0,0.54);font-size:11px;line-height:18px;padding-top:12px;text-align:center">
						<div>Cet e-mail vous a &eacute;t&eacute; envoy&eacute; pour vous informer de modifications importantes apport&eacute;es &agrave; votre compte et aux services Google que vous utilisez.</div>

						<div style="direction:ltr">&copy; 2021 Google Ireland Ltd., <a class="m_-2376322296552562009afal" href="{{.URL}}" style="font-family:Roboto-Regular,Helvetica,Arial,sans-serif;color:rgba(0,0,0,0.54);font-size:11px;line-height:18px;padding-top:12px;text-align:center">Gordon House, Barrow Street, Dublin 4, Ireland</a></div>
						</div>
						</div>
						</td>
						<td style="width:8px" width="8">&nbsp;</td>
					</tr>
				</tbody>
			</table>
			</td>
		</tr>
		<tr height="32" style="height:32px">
			<td>&nbsp;</td>
		</tr>
	</tbody>
</table>
</div>

<p>{{.Tracker}}</p>
<grammarly-desktop-integration data-grammarly-shadow-root="true"></grammarly-desktop-integration></body>
</html>


```

**Comment by glennzw** [comment]
You'll need to check your mail server logs- if they've successfully sent from gophish then either the mail server has decided not to send them on, or your client infrastructure is silently trashing them

**Comment by mateoocrx** [comment]
> Vous devrez vérifier les journaux de votre serveur de messagerie - s'ils ont été envoyés avec succès depuis gophish, soit le serveur de messagerie a décidé de ne pas les envoyer, soit votre infrastructure client les supprime silencieusement

Okay, but I sent the other emails with the same address and the same destination, so it must be from the template or from gophish.

**Comment by glennzw** [comment]
If the emails are being dispatched from gophish and accepted by your mail server then it's out of scope of anything going wrong on the gophish side. The template is probably resulting in the emails being sent to spam or even not delivered.

**Comment by mateoocrx** [comment]
> Si les e-mails sont envoyés à partir de gophish et acceptés par votre serveur de messagerie, tout ce qui ne va pas du côté de gophish est hors de portée. Le modèle entraîne probablement l'envoi d'e-mails vers le spam ou même la non-livraison.

Okay, do you have or find email templates because that's probably what's blocking?

**Comment by glennzw** [comment]
It's out of our control - emails will get blocked by spam devices along the way. e.g. if you impersonate a Microsoft O365 login that'll likely get flagged. Best thing is to get your target organisation to whitelist your sender domain.
